src/examples/avatar/AvatarStatus.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { PALETTE } from '@zendeskgarden/react-theming';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => {
+  const iconColor = PALETTE.grey[600];
+
+  return (
+    <Grid>
+      <Row alignItems="center">
+        <Col textAlign="center">
+          <Avatar backgroundColor={iconColor} status="away">
+            <img
+              alt="away avatar"
+              src="https://garden.zendesk.com/react-components/avatars/images/avatar-3.png"
+            />
+          </Avatar>
+        </Col>
+        <Col textAlign="center">
+          <Avatar backgroundColor={iconColor}>
+            <img
+              alt="standard avatar"
+              src="https://garden.zendesk.com/react-components/avatars/images/avatar-3.png"
+            />
+          </Avatar>
+        </Col>
+        <Col textAlign="center">
+          <Avatar backgroundColor={iconColor} status="available">
+            <img
+              alt="available avatar"
+              src="https://garden.zendesk.com/react-components/avatars/images/avatar-3.png"
+            />
+          </Avatar>
+        </Col>
```

Comment:
Treatment still not right. Should be away (yellow), available (green), active (red badge). Update alt text to match.